minikube.mk

```diff
@@ -47,6 +47,6 @@ minikube-start: minikube-start-$(CONTAINER_RUNTIME)
 .PHONY: phony_explicit
 minikube-start-%: minikube-install
 	$(MINIKUBE) status -p minikube-$* -f {{.APIServer}} >/dev/null || \
-	$(MINIKUBE) start -p minikube-$* --driver=$(MINIKUBE_DRIVER) --kubernetes-version=$(KUBERNETES_VERSION) --container-runtime=$* --wait=all
+	$(MINIKUBE) start -p minikube-$* --driver=$(MINIKUBE_DRIVER) --kubernetes-version=$(KUBERNETES_VERSION) --container-runtime=$* --wait=all $${MINIKUBE_PARAMS}
```

Comment:
IIUC, we are not using `minikube-start` anymore on the CI, so why is this needed?

Reply:
We are using it in the [action](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/actions/setup-minikube/action.yml#L28) now and [`MINIKUBE_PARAMS`](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/actions/setup-minikube/action.yml#L33) are used to pass custom config to minikube. Does that make sense?

Reply:
Aahh we are using `minikube-start` in the action. I thought we were using `manusa/actions-setup-minikube@v2.7.2`. Go it now. Thanks!